src/store/d-apps/d-apps-epics.ts

```diff
@@ -142,6 +151,36 @@ const approveOperationRequestEpic = (action$: Observable<Action>) =>
     )
   );
 
+const waitForOperationCompletionEpic = (action$: Observable<Action>) =>
+  action$.pipe(
+    ofType(waitForOperationCompletionAction),
```

Comment:
could you create signle action instead of `waitForInternalOperationCompletionAction` and `waitForOperationCompletionAction`?